models/wrf_hydro/noah_hydro_mod.f90

```diff
@@ -1080,25 +1086,28 @@ end subroutine get_routelink_constants
 !-----------------------------------------------------------------------
 !> 
 
-subroutine fill_connections(toIndex,fromIndices,fromIndsStart,fromIndsEnd)
+subroutine fill_connections(toIndex,fromIndices,fromIndsStart,fromIndsEnd,num_up_links)
```

Comment:
in this pull request, you've made num_up_links a module variable (like connections is) so it is available to fill_connections without being passed in.

Reply:
Good catch. Done!